configs/namespaces.go

```diff
@@ -16,6 +16,17 @@ const (
 	NEWUSER NamespaceType = "NEWUSER"
 )
 
+func NamespaceTypes() []NamespaceType {
```

Comment:
Probably becuase exported vars in go can be mutabed while the return of the a function cannot